app/api/helpers/scheduled_jobs.py

```diff
@@ -155,6 +156,16 @@ def expire_pending_tickets():
         db.session.commit()
 
 
+def delete_ticket_holders_no_order_id():
+    from app import current_app as app
+    with app.app_context():
+        order_expiry_time = get_settings()['order_expiry_time']
+        TicketHolder.query.filter(TicketHolder.order_id == None, TicketHolder.deleted_at == None,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comparison to None should be 'if cond is None:'](https://app.codacy.com/app/fossasia/open-event-orga-server/pullRequest?prid=4674573)